intel_extension_for_transformers/neural_chat/config.py

```diff
@@ -312,12 +312,51 @@ class FinetuningArguments:
     )
 
 @dataclass
-class FinetuningConfig:
+class TTSDatasetArguments:
+    audio_paths: Optional[str] = field(default=None, metadata={"help": "The path of audios."})
+    gender: Optional[str] = field(default=None, metadata={"help": "Gender."})
+    language: Optional[str] = field(default="English", metadata={"help": "Language."})
+
+@dataclass
+class TTSModelArguments:
+    step: int = field(default=0, metadata={"help": "TTS model step."})
+    warmup_step: int = field(default=0, metadata={"help": "TTS model warmup step."})
+    learning_rate: float = field(default=5e-5, metadata={"help": "Learning rate."})
+ 
+@dataclass
+class BaseFinetuningConfig:
+    model_args: ModelArguments
+    data_args: DataArguments
+    training_args: TrainingArguments
+    finetune_args: FinetuningArguments
+
+@dataclass
+class TextGenerationFinetuningConfig(BaseFinetuningConfig):
+    model_args: ModelArguments

```

Comment:
TextGenerationFinetuningConfig, SummarizationFinetuningConfig and CodeGenerationFinetuningConfig all have the same config with BaseFinetuningConfig?

Reply:
yes, all these tasks follow the same training paradigm, just with different prompt templates which are predefined.